### PR TITLE
spatial libs docs page reorg

### PR DESCRIPTION
**Pull request by darkblue-b**
minor edit - en only

### CODE REVIEW

**Comment by robe2** [comment]
I would also change:

**Available from prior OSGeo-Live releases:**

to 

**Available in prior OSGeo-Live releases:**

**from** suggests it was simply introduced in a prior release and still maintained.  In is more a we don't include it anymore.

As far as the case of GEOS, JTS, and GeoTools, I feel like there should be a separate categorization for those since you are just removing the quick start documentation and not the library.

Perhaps a subsection called something like:  Reorg

**Projects moved to Library section, quicks starts no longer included**

**Comment by cvvergara** [comment]
@robe2 
BTW, they never had a quickstart

**Comment by camerons** [comment]
@darkblue-b I see that you have re-included GEOS, JTS, and GeoTools into the main documentation. I had moved these docs into the retired section of our docs based on what I believe we agreed in IRC meeting discussions.
Justification for retiring is: Our volunteers are not keeping up with the maintenance of these library documents, and they don't have quickstarts, and our focus should primarily be on new users rather than developers. 
I'm open to using words like "project overview not maintained from version 10.5" but I don't want to suggest that we are keeping the docs up to date when we are not.

**Comment by darkblue-b** [comment]
the JTS docs are available right now, and .. JTS docs are important.  I continue support this change. Minor edits coming per robe2

**Comment by jodygarnett** [comment]
This is a tricky one - if OSGeo live is primarily an outreach tool for GIS Professionals - then it is hard to include information for software developers.

That said these tools are the "geospatial heart" of our community, providing the power behind many of the runnable tools enjoyed by the GIS Professionals. It will be hard to explain to these communities why they are not included.

For GeoTools, if you are including OpenJDK, you are welcome to include the Quickstart application (which allows you create and run a simple shapefile viewer).

For JTS please include the jts test runner, it is great and allows a visual of what JTS can do. I use it often for teaching postgis since it explains the operations visually.

TLDR: If quickstarts are all that you are missing it can be solved; if this is instead a strategic choice to focus on GIS professionals then I will understand.

**Comment by cvvergara** [comment]
The projects in debate:

Have an overview:
http://adhoc.osgeo.osuosl.org/livedvd/docs/en/overview/geos_overview.html
http://adhoc.osgeo.osuosl.org/livedvd/docs/en/overview/jts_overview.html
http://adhoc.osgeo.osuosl.org/livedvd/docs/en/overview/geotools_overview.html

Those are libraries that are/can-be   used in other projects.
The difficulty level of a quickstart on a library can vary.

## My perspective as a library developer

I will compare with what I know (pgRouting):
pgRouting is a library, and "normally" its used linked to a postgreSQL database. This quickstart is not so difficult, because using the library is done by using postgreSQL.

Now, lets imagine that, it can be used only "detached" from the postgreSQL database (which BTW it is possible, because its a library)
The  quickstart that is "required" : 
Its not easy to do, the simplest problem I can think of, needs lots of work, lots of dependencies etc. (basically teach how to create a C++ program and link pgRouting functionality to that program, dependencies, compile, link etc).
In this case I would rather keep only the overview, and send the readers to the project's page, for further reading. Mainly because the quick start would not be quick at all.

**Comment by robe2** [comment]
Yes agreed let's not make quick starts that aren't quick ::)

**Comment by jodygarnett** [comment]
I think some of the point here is if they want to try out jts and geotools they can run GeoServer, or uDig. If they want to try out GEOS they can try out QGIS or PostGIS or ...

Thinking: I bet you could make a GEOS quickstart as a QGIS Python script, or maybe just make a python script.

For reference here is the GeoTools quickstart:

```
package org.geotools.tutorial.quickstart;

import java.io.File;

import org.geotools.data.FileDataStore;
import org.geotools.data.FileDataStoreFinder;
import org.geotools.data.simple.SimpleFeatureSource;
import org.geotools.map.FeatureLayer;
import org.geotools.map.Layer;
import org.geotools.map.MapContent;
import org.geotools.styling.SLD;
import org.geotools.styling.Style;
import org.geotools.swing.JMapFrame;
import org.geotools.swing.data.JFileDataStoreChooser;

public class Quickstart {
    public static void main(String[] args) throws Exception {
        File file = JFileDataStoreChooser.showOpenFile("shp", null);
        if (file == null) {
            return;
        }

        FileDataStore store = FileDataStoreFinder.getDataStore(file);
        SimpleFeatureSource featureSource = store.getFeatureSource();

        MapContent map = new MapContent();
        map.setTitle("Quickstart");
        
        Style style = SLD.createSimpleStyle(featureSource.getSchema());
        Layer layer = new FeatureLayer(featureSource, style);
        map.addLayer(layer);

        JMapFrame.showMap(map);
    }
}
```

**Comment by robe2** [comment]
I don't really think we need a Quickstart for GEOS.  Sounds like work to keep up to date and certainly not something I have the energy to write up and maintain and @strk is already exhausted with other work.  I suspect @mloskot  is not interested in writing one either, though I may be wrong.

To me if it's a QGIS Python script then it's pyQGIS quick start not a GEOS quickstart and then a python only  quickstart is not something someone using the PHP bindings or using the C-API can use directly so much of the audience is lost.  Besides they'd probably be better served using something like Shapely, https://github.com/Toblerity/Shapely if they wanted to use GEOS in python.

So they might as well just look at the GEOS regress tests or be directed to code in PostGIS / QGIS / SpatiaLite / GDAL that uses the GEOS C-API bindings for something useful they can emulate.

Thanks,
Regina

**Comment by kalxas** [comment]
I agree with keeping the overviews of those important libraries, but we will need to bring them up to date.

**Comment by kalxas** [comment]
I think we need to vote on mailing list for this PR

**Comment by darkblue-b** [comment]
This simple change only includes what was **already** included regarding three libraries -- **JTS**, **GEOS**, and **GeoTools**.  The corresponding Overview pages have been updated and checked.

**Comment by mloskot** [comment]
@robe2 re GEOS docs - no, not really. IMHO, source code of GEOS test will beat any documentation one can write for GEOS in reasonable amount of time.

**Comment by kalxas** [comment]
@darkblue-b can you please fix the conflict so we can merge?

**Comment by cvvergara** [comment]
I will fix them, has conflicts with the changes of the small icon on overview files
